athina/evals/ragas/answer_relevancy/evaluator.py

```diff
@@ -0,0 +1,66 @@
+from typing import List
```

Comment:
Can we link to the RAGAS source evaluator Github and document here?

Reply:
Done